### PR TITLE
Scripts that setup the Test Runner

### PR DESCRIPTION
**Pull request by jsconan**
Add scripts to setup the old or the new Test Runner.

### CODE REVIEW

File: scripts/install/SetOldTestRunner.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        $itemQtiExt = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiItem');
+        $compilerClassConfig = 'oat\\taoQtiItem\\model\\QtiItemCompiler';
+        $itemQtiExt->setConfig('compilerClass', $compilerClassConfig);
+
+        $testQtiExt = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiTest');
+        $testRunnerConfig = $testQtiExt->getConfig('testRunner');
+        $testRunnerConfig['test-session'] = '\\taoQtiTest_helpers_TestSession';
+        $testQtiExt->setConfig('testRunner', $testRunnerConfig);
+
+        return new \common_report_Report(\common_report_Report::TYPE_SUCCESS, 'New test runner activated');
```
Comment (antoinerobin) [inline scripts/install/SetOldTestRunner.php]: please modify the report message

Comment (jsconan) [inline scripts/install/SetOldTestRunner.php]: Nice catch!

Changes made

File: scripts/install/SetOldTestRunner.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * 
+ * @package oat\taoQtiTest\scripts\install
+ */
+class SetOldTestRunner extends \common_ext_action_InstallAction
+{
+    public function __invoke($params)
+    {
+        $deliveryExt = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoDelivery');
+        $deliveryServerConfig = $deliveryExt->getConfig('deliveryServer');
+        $deliveryServerConfig->setOption('deliveryContainer', 'oat\\taoDelivery\\helper\\container\\DeliveryClientService');
```
Comment (antoinerobin) [inline scripts/install/SetOldTestRunner.php]: the deliveryContainer should rather be ```oat\taoDelivery\helper\container\DeliveryServiceContainer```

Comment (jsconan) [inline scripts/install/SetOldTestRunner.php]: woops!

**Pull request merged**